### PR TITLE
data: DaduDev

### PR DESCRIPTION
**Pull request by DaduDev**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5035"></a>

### CODE REVIEW

**Review by Ananya2001-an — CHANGES_REQUESTED** [review]
You have to create this file inside the data folder. Also remove the `avatar` and `displayPublicStats` keys from the JSON object since according to new upgrades we don't need them anymore.

**Comment by DaduDev** [comment]
> You have to create this file inside the data folder. Also remove the `avatar` and `displayPublicStats` keys from the JSON object since according to new upgrades we don't need them anymore.

Thank you

File: data/DaduDev.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Dadudev1",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://twitter.com/ig.dadu",
+      "icon": "FaTwitter"
```
Comment (Panquesito7) [inline data/DaduDev.json:14]: You might want to use the Instagram icon here instead.
```suggestion
      "icon": "FaInstagram"
```

**Review by SurajPratap10 — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**